docs/rules/html-indent.md

```diff
@@ -0,0 +1,115 @@
+# enforce consistent indentation in `<template>` (vue/html-indent)
+
+## :book: Rule Details
+
+This rule enforces a consistent indentation style in `<template>`. The default style is 4 spaces as same as [the core indent rule](http://eslint.org/docs/rules/indent).
+
+- This rule checks all tags, also all expressions in directives and mustaches.
+- In the expressions, this rule supports ECMAScript 2017 syntaxes. It ignores unknown AST nodes, but it might be confused by non-standard syntaxes.
+
+:-1: Examples of **incorrect** code for this rule:
+
+```html
+<template>
+ <div class="foo">
+   Hello.
+    </div>
+</template>
+```
+
+:+1: Examples of **correct** code for this rule:
+
+```html
+<template>
+    <div class="foo">
+        Hello.
+    </div>
+</template>
+```
+
+```html
+<template>
+    <div class="foo">
+        Hello.
+    </div>
+    <div
+        id="a"
+        class="b"
+        :other-attr="{
+            aaa: 1,
+            bbb: 2
+        }"
+        @other-attr2="
+            foo();
+            bar();
+        "
+    >
+        {{
+            displayMessage
+        }}
+    </div>
+</template>
+```
+
+## :wrench: Options
+
+```json
+{
+    "vue/html-indent": ["error", type, {
+        "attribute": 1,
+        "closeBracket": 0,
+        "switchCase": 0,
```

Comment:
could you add an example for the `switchCase` option?

Reply:
Or maybe I should remove the option from this rule since `switch` is [a prohibited keyword](https://github.com/vuejs/vue/blob/c0da43d22f8f4b9aeb4f49d4e86cd9704daaff3f/src/compiler/error-detector.js#L7-L11).